utils/get-bundle.js

```diff
@@ -17,8 +18,11 @@ const defaultRoot = getDefaultRoot();
 //   const webpackConfigPath = path.join(root, 'webpack.config.js');
 // };
 
+const getId = query => md5(query);
```

Comment:
memoization for this case would be awesome.
So after each new request for `getBestBundleDataFromConfig `, we can avoid md5 call for same queries.

Something like
```js
const cache = {};
const getId = query => cache[query] || (cache[query] =  md5(query));
```

Reply:
done